src/main/java/com/example/codebase/domain/notification/dto/NotificationResponse.java

```diff
@@ -0,0 +1,96 @@
+package com.example.codebase.domain.notification.dto;
+
+import com.example.codebase.controller.dto.PageInfo;
+import com.example.codebase.domain.notification.entity.Notification;
+import com.example.codebase.domain.notification.entity.NotificationType;
+import com.example.codebase.domain.notification.entity.NotificationWithIsRead;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.domain.Page;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class NotificationResponse {
+
+    @Getter
+    @Setter
+    @Schema(description = "알림 개별 응답")
+    public static class Get {
+        private Long notificationId;
+
+        @JsonFormat
+        private String message;
+
+        private String type;
+
+        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
+        private String createdTime;
+
+        private Boolean isRead;
+
+        public static Get from(NotificationWithIsRead notification) {
+            Get response = new Get();
+            response.notificationId = notification.getNotification().getNotificationId();
+            response.message = notification.getNotification().getMessage();
+            response.type = notification.getNotification().getType().name();
+            response.createdTime = notification.getNotification().getCreatedTime().toString();
+            response.isRead = notification.getIsRead();
+            return response;
+        }
+    }
+
+    @Getter
+    @Setter
+    public static class GetAll {
+        private List<Get> notifications;
+
+        private PageInfo pageInfo;
+
+        public static GetAll from(Page<NotificationWithIsRead> notifications) {
+            GetAll response = new GetAll();
+            response.pageInfo = PageInfo.from(notifications);
+            response.notifications = notifications.stream()
+                    .map(NotificationResponse.Get::from)
+                    .toList();
+            return response;
+        }
+    }
+
+    @Getter
+    @Setter
+    @Schema(description = "알림 이벤트 응답")
```

Comment:
EventMessage는 중복되지 않는 DTO 클래스 명이지만 추후 의미가 여러 곳에서 사용될 가능성이 있어서 다음과 같이 제안드립니다~

```suggestion
    @Schema(name = "NotifcationResponse.EventMessage", description = "알림 이벤트 응답")
```